app/src/main/java/com/sdp/movemeet/view/chat/MessageViewHolder.java

```diff
@@ -79,12 +79,12 @@ public void bindMessage(Message message, ChatActivity chatActivity) {
      * @param message Message object containing the message data (in this case the message data of
      *                interest is the URL of the image)
      */
-    private void handlingUserProfilePicture(Message message) {
+    private void handlingUserProfilePicture(Message message, ChatActivity chatActivity) {
```

Comment:
Why the chatActivity is add in this function ?

Reply:
It's necessary as the image handler uses the activity it loads the image to (we need the context to check the permissions and get access to the progress bar).